content/id/docs/tutorials/kubernetes-basics/explore/explore-intro.html

```diff
@@ -0,0 +1,143 @@
+---
+title: Melihat Pod dan Node
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="id">
+
+<body>
+
+<link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+     <div class="col-md-8">
+          <h3>Tujuan</h3>
+                <ul>
+                    <li>Belajar tentang Pod Kubernetes.</li>
+                    <li>Belajar tentang Node Kubernetes.</li>
+                    <li>Mengatasi masalah pada aplikasi-aplikasi yang di-<i>deploy</i>.</li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h2>Pod Kubernetes</h2>
+                <p>Ketika kamu membuat sebuah Deployment di Modul <a href="/id/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/">2</a>, Kubernetes membuatkan sebuah <b>Pod</b> untuk menempatkan instans aplikasimu. Pod merupakan sebuah abstraksi Kubernetes yang merepresentasikan sebuah grup yang terdiri dari satu atau lebih kontainer (seperti Docker), dan beberapa sumber daya bersama untuk kontainer-kontainer itu. Sumber daya tersebut termasuk:</p>
```

Comment:
Container is API name, no need to translate. https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/
```suggestion
                <p>Ketika kamu membuat sebuah Deployment pada Modul <a href="/id/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/">2</a>, Kubernetes membuatkan sebuah <b>Pod</b> untuk menempatkan _instance_ aplikasimu. Pod merupakan sebuah abstraksi Kubernetes yang merepresentasikan sebuah grup yang terdiri dari satu atau lebih Container (seperti Docker), dan beberapa sumber daya bersama untuk Container-Container itu. Sumber daya tersebut termasuk:</p>
```

Reply:
I think the containers that used here are not referencing into Container in API Object. Because in English it originally using lowercase. The word "container" is also translated as "kontainer" in `/id/docs/tutorials/kubernetes-basics/`.

Reply:
in RSNI3 there is `instance = instans`